images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -111,7 +94,22 @@ Push-Location -Path $sdk.FullName
     "add-ons;addon-google_apis-google-21" `
     "cmake;3.6.4111459" `
     "cmake;3.10.2.4988404" `
-    "patcher;v4"
+    "patcher;v4" `
+    "ndk-bundle"
+
+    # Android NDK root path.
+    $ndk_root = "C:\Program Files (x86)\Android\android-sdk\ndk-bundle"
+
+    if(Test-Path $ndk_root){
```

Comment:
Could you please fix style a bit to be consistent with other code:
```
if (Test-Path $ndk_root) {
...
} else {
}
```
Mostly spaces and new line symbols

Reply:
thank you, I have changed this.